docs/auth0_actions_create.md

```diff
@@ -7,12 +7,26 @@ Create a new action.
 
 To create interactively, use `auth0 actions create` with no flags.
 
-To create non-interactively, supply the action name, trigger, secrets and dependencies through the flags.
+To create non-interactively, supply the action name, trigger, code, secrets and dependencies through the flags.
 
+## Usage
```

Comment:
Added a "Usage" header to make it stand out more and to be more similar to the terminal help output.